openreview/conference/templates/programchairWebfield.js

```diff
@@ -1227,6 +1310,10 @@ controller.addHandler('areachairs', {
 
       var uniqueIds = _.union(uniqueReviewerIds, uniqueAreaChairIds);
 
+      if (pcAssignmentTagInvitations && pcAssignmentTagInvitations.length) {
+        getPcAssignmentTags();
```

Comment:
this is an async function and it should be chained with the other promises.

Reply:
@zbialecki I made some improvements  using the network calls, do you mind to take a look?